Content.Client/UserInterface/Systems/Ghost/Controls/Roles/GhostRolesWindow.xaml.cs

```diff
@@ -18,14 +20,88 @@ public void ClearEntries()
             EntryContainer.DisposeAllChildren();
         }
 
+        public IEnumerable<Collapsible> GetAllCollapsibleBoxes()
+        {
+            return EntryContainer.Children.OfType<Collapsible>();
+        }
+
+        public Collapsible? GetCollapsibleById(string id)
+        {
+            return EntryContainer.Children
+                .OfType<Collapsible>()
+                .FirstOrDefault(c => c.Name == id);
+        }
+
+        public Dictionary<int, bool> SaveCollapsibleBoxesStates()
+        {
+            var collapsibleStates = new Dictionary<int, bool>();
```

Comment:
Make this a `HashSet<>` instead where an element being present means it's collapsed.

Reply:
Done in [2a8376f](https://github.com/space-wizards/space-station-14/pull/32717/commits/2a8376f1feef6cae60bc06b2c42af9c3f107c06b)